sessionctx/stmtctx/stmtctx.go

```diff
@@ -423,3 +426,45 @@ func (sc *StatementContext) ShouldIgnoreOverflowError() bool {
 	}
 	return false
 }
+
+// CopTasksDetails returns some useful information of cop-tasks during execution.
```

Comment:
Is this related with this PR?

Reply:
My fault, it's a mistake.
I will fix it and cherry-pick to 2.1 after merging.

Reply:
Fixed.